### PR TITLE
{math}[GCC/13.2.0] bliss v0.77

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bliss/bliss-0.77_install_fix.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff -u CMakeLists.txt.orig CMakeLists.txt
```
Comment (boegel) [inline easybuild/easyconfigs/b/bliss/bliss-0.77_install_fix.patch:1]: @PetrKralCZ Please add a short description at the top of the patch + mention author or source (and fix checksum accordingly)

File: easybuild/easyconfigs/b/bliss/bliss-0.77-GCC-13.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+checksums = [
+    {'bliss-0.77.zip': 'acc8b98034f30fad24c897f365abd866c13d9f1bb207e398d0caf136875972a4'},
+    {'bliss-0.77_install_fix.patch': '01a0824c5b121bd6ab0df324ee9c9dd4d4f0bfd8bb2800157cf270e8a1b194ea'},
+]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+
+configopts = "-DUSE_GMP=OFF "
```
Comment (boegel) [inline easybuild/easyconfigs/b/bliss/bliss-0.77-GCC-13.2.0.eb:23]: Why not add `GMP` as a dependency, and use `ON`? Any downsides to that?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/214aef6e04360cce860c44e1a22aed3e for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**